projects/simple-if-else-dictionary/app.py

```diff
@@ -0,0 +1,30 @@
+#Simple if else Dictionary.
+import json
+from difflib import get_close_matches	#if you enter this library will check it.
+data = json.load(open("data.json"))
+
+def translate(word):
+    word = word.lower()
+    if word in data:
+        return data[word]
```

Comment:
don't use multiple return statement in function
store result in some variable and use these variables to return an output value
```suggestion
         result = data[0]
        return result
```